compiler/luci/import/src/Importer.cpp

```diff
@@ -274,7 +473,8 @@ std::unique_ptr<loco::Graph> Importer::import(const circle::Model *model) const
     return nullptr;
 
   // Convert circle::Model to loco::Graph
-  convert_graph(*source_ptr, reader, graph.get());
+  // not copy constants if model's buffer provided
+  convert_graph(*source_ptr, reader, graph.get(), _model_buffer == nullptr);
```

Comment:
I cann't check existing `convert_graph` is not used anymore from this change. is it not used anymore?

Reply:
Yes, it not used anymore =)

Checked with removing of old `convert_graph`: project build finishes successfully.

Reply:
OK, then we can change existing one to your new one.